src/redux/backend-session.ts

```diff
@@ -222,6 +224,19 @@ const registerMux = (language: Language, dclId: string) => {
   });
 };
 
+const selectImageTag = (language: Language, version: string) => {
+  if(language == "python"){
+    const python_versions = ["3.5", "3.6", "3.7"];
+    const default_version = "3.5";
+    if(version && python_versions.includes(version)) {
```

Comment:
I suggest to only use `python_versions.includes(version)` (I don't see how `version &&` could be useful)